core/sail/nativerdf/src/main/java/org/eclipse/rdf4j/sail/nativerdf/ValueStore.java

```diff
@@ -256,7 +255,7 @@ public int getID(Value value) throws IOException {
 					// Store id in cache
 					NativeValue nv = getNativeValue(value);
 					nv.setInternalID(id, revision);
-					valueIDCache.put(nv, new Integer(id));
+					valueIDCache.put(nv, Integer.valueOf(id));
```

Comment:
This has nothing to do with the issue, but I stumbled across this while checking `NativeIRI`.